torch/_torch_docs.py

```diff
@@ -152,6 +152,34 @@ def merge_dicts(*dicts):
     tensor([ 1.2294,  2.2004,  1.3690,  1.7298])
 """.format(**common_args))
 
+add_docstr(torch.acosh,
+           r"""
+acosh(input, out=None) -> Tensor
+
+Returns a new tensor with the inverse hyperbolic cosine of the elements of :attr:`input`.
+The domain of the inverse hyperbolic cosine is `[1, inf)` and values outside this range
```

Comment:
Cool!. I forgot to mention it earlier (my mistake) but let's put this sentence into a Note. See torch.bincount, for example. (It's very simple.)

Reply:
Done, thanks Ruberry. :)